src/lib/utils.ts

```diff
@@ -8,3 +8,16 @@ export function toArray<T>(toConvert: T | T[]): T[] {
   const toReturn = toConvert instanceof Array ? toConvert : [toConvert];
   return toReturn.filter((value: T) => !!value);
 }
+
+export const ensureFormDataHasKeys = (
+  formData: Record<string, string>,
+  ...keys: string[]
+): Record<string, string> => {
+  keys.forEach((key: string) => {
+    if (!formData[key]) {
+      formData[key] = "";
+    }
+  });
+
+  return formData;
```

Comment:
`ensureFormDataHasKeys()` is side-effecting as it mutates `formData`.  This makes for more pleasant reading on the pages that will use it, but may result in annoying bugs if future developers aren't aware of the side effect.

Option to change this to be a pure function.

Reply:
Agreed, I am happy for this to be updated to a pure function.  It's returning the `formData` even though it doesn't need to as it mutates the data on the record.

Also, love seeing a record.

Reply:
Done.  Love a good Record.